classes/submission/maps/Schema.php

```diff
@@ -526,16 +526,39 @@ protected function canChangeMetadata(Enumerable $stageAssignments, Submission $s
     /**
      * Get details about the review assignments for a submission
      */
-    protected function getPropertyReviewAssignments(Enumerable $reviewAssignments, bool|Collection $anonymizeReviews = false): array
+    protected function getPropertyReviewAssignments(Enumerable $reviewAssignments, bool|Collection $anonymizeReviews = false, Submission $submission, array $stages): array
     {
-        $reviews = [];
         $request = Application::get()->getRequest();
         $currentUser = $request->getUser();
+    
+        // application specific review stages
+        $reviewStageIds = [WORKFLOW_STAGE_ID_EXTERNAL_REVIEW];
+        if (Application::get()->getName() === 'omp') {
```

Comment:
It's better to find or modify existing methods to retrieve review stages. E.g., I see the protected method in the [submission collector](https://github.com/pkp/pkp-lib/blob/9ac7230eeaf311d528f8d2e359921e0d2fba9884/classes/submission/Collector.php#L916). We could make it public and move to the PKPApplication/Application class.